packages/client/ui/react-ui/src/components/auth/methods/email/EmailSignIn.tsx

```diff
@@ -93,19 +112,56 @@ export function EmailSignIn({ setOtpEmailData }: { setOtpEmailData: (data: OtpEm
                                     }}
                                 />
                             )}
-                            {!emailError && !isLoading && (
+                            {showGoogleContinueButton ? (
+                                <button
```

Comment:
this feels prime for a separate component so the oauthlistener hook is kept out side of this flow